src/controllers/message.controller.js

```diff
@@ -0,0 +1,45 @@
+import { Message } from '../models/Message.model.js';
+import { EventEmitter } from 'events';
+
+const MESSAGE = 'message';
+
+export const emitter = new EventEmitter();
+
+export const getMessages = async (req, res) => {
+  const whereClauses = {};
+  const { roomId } = req.query;
+
+  whereClauses.roomId = roomId;
+  res.send(await Message.findAll({ where: whereClauses }));
+};
+
+export const postMessage = async (req, res) => {
+  const { text, author, roomId } = req.body;
+
+  if (!text) {
+    res.status(400).send({ message: 'Text message is missing' });
+  }
+
+  if (!author) {
+    res.status(400).send({ message: 'Author of text is missing' });
+  }
+
+  if (!roomId) {
+    res.status(400).send({ message: 'Room is missing' });
+  }
```

Comment:
The current implementation sends a response immediately if any of the fields 'text', 'author', or 'roomId' are missing. This will terminate the function execution, which is correct. However, consider using 'return' statements after sending the response to make it explicit that the function should not continue executing.